misc/python/materialize/parallel_workload/action.py

```diff
@@ -1610,6 +1620,7 @@ def __init__(
 write_action_list = ActionList(
     [
         (InsertAction, 50),
+        (SelectOneAction, 50),
```

Comment:
SelectOneAction is not a `write` action, is it? Also, the probability to run it seems quite high.

Reply:
Correct, but mixing selects and inserts in a single transaction (I was not even aware this was possible) is the only way to trigger the panic from https://github.com/MaterializeInc/materialize/issues/24341